.cargo/config.toml

```diff
@@ -1,7 +1,7 @@
 # paths = ["/path/to/override"] # path dependency overrides
 
 [alias] # command aliases
-
+install_soroban = "install --version 21.0.0-rc.1 --root ./target soroban-cli --debug"
```

Comment:
@willemneal is there a way to do something like [the js-stellar-sdk `test/e2e/initialize.sh` script](https://github.com/stellar/js-stellar-sdk/blob/840f57da0d2fe3cc52d37670becd76ecd8b2cdca/test/e2e/initialize.sh#L17-L30) for our integration tests? Before the integration tests run, I would like to make sure that the correct version of soroban is pinned, and maybe also run the `soroban contract init` command that's currently being used to build the `soroban-init-boilerplate` (see below).

Reply:
That looks great! Added an issue:

- https://github.com/loambuild/loam-sdk/issues/68

For this PR, I noticed that we already have `just setup` that installs soroban-cli, and I updated that version to match the one that this PR relies on.